generated/ApiResponse.h

```diff
@@ -0,0 +1,47 @@
+#ifndef ORG_OPENAPITOOLS_MODEL_ApiResponse_H_
+#define ORG_OPENAPITOOLS_MODEL_ApiResponse_H_
+
+#include "serialization.h"
+
+#include <optional>
+
+#include <string>
+
+namespace org {
+namespace openapitools {
+namespace model {
+
+/**
+ * \brief Describes the result of uploading an image resource
+ */
+struct ApiResponse {
+    static const char *dtoName() { return "swagger::dto::ApiResponse"; }
+
+
+
+    // optional fields
+    std::optional<int32_t> code;
+    std::optional<std::string> type;
+    std::optional<std::string> message;
+};
+
+void to_json(nlohmann::json& j, const ApiResponse& obj) {
+    j = nlohmann::json{
+        {"code", obj.code},
+        {"type", obj.type},
+        {"message", obj.message},
```

Comment:
I suppose `nlohmann::json` can deal with `std::optional`? ... asking for a friend.

Reply:
See below